src/samples/flow/contiki-demo/contiki-demo.fbp

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env sol-fbp-runner
+# This file is part of the Soletta Project
+#
+# Copyright (C) 2015 Intel Corporation. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+#   * Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#   * Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in
+#     the documentation and/or other materials provided with the
+#     distribution.
+#   * Neither the name of Intel Corporation nor the names of its
+#     contributors may be used to endorse or promote products derived
+#     from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+# OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# This is a trash disposer that was write to show Soletta working on a board
+# running Contiki.
+# The behavior here is pretty simple, you press the button it make the servo
+# motors to spin 180 degrees, causing the trash being thrown away, after
+# 3 seconds it return to the initial position.
+
+base_servo(servo-motor/controller:chip=0,duty_cycle_range=min:1000|max:2000,period=20000,pin=0)
+top_servo(servo-motor/controller:chip=0,duty_cycle_range=min:1000|max:2000,period=20000,pin=1)
+button(gpio/reader:active_low=false,edge_falling=true,edge_rising=true,pin=15)
+led(gpio/writer:active_low=false,pin=16)
```

Comment:
IMO we should have this file not named after contiki, just have a contiki+board ".conf" file and make it also run on desktop with another conffile using gtk and we may provide one for galileo+linux using some of our reference sensors.

Then we have one more test to show ".conf" capabilities.

What we could do, and here @dorileo may be of help, is to get this to be converted to C and compiled only if a certain ".conf" exists.

Say we declare a `BOARD_CONFFILE` in our Kconfig and Makefile. This FBP->C would rely on existence of `BOARD_CONFFILE` in the FBP source directory (if relative) or an absolute path. If that conffile doesn't exist, this sample is skipped.

This allow this test to be compiled and tested on other OS and boards such as Galileo with Groove Starter Kit.